src/plugins/dashboard/public/dashboard_app/top_nav/ignore_filter_tour.tsx

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import React, { useState } from 'react';
+import {
+  EuiButtonEmpty,
+  EuiCheckbox,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiLink,
+  EuiText,
+  EuiTourStep,
+} from '@elastic/eui';
+import type { DataView } from '@kbn/data-views-plugin/public';
+import useMount from 'react-use/lib/useMount';
+import { pluginServices } from '../../services/plugin_services';
+import { filterTourStrings } from '../_dashboard_app_strings';
+
+const FILTER_TOUR_OPT_OUT_LOCAL_KEY = 'dashboardHideFilterTour';
+
+export const IgnoreFilterTour = ({ dataViews = [] }: { dataViews: DataView[] }) => {
+  const [isTourOpen, setTourOpen] = useState(false);
+  const [hideTour, setHideTour] = useState(false);
+  const tourOptOut = localStorage.getItem(FILTER_TOUR_OPT_OUT_LOCAL_KEY) === 'true';
+
+  const {
+    application: { navigateToApp },
+  } = pluginServices.getServices();
+
+  useMount(() => {
+    if (!tourOptOut) {
+      setTourOpen(true);
+    }
+  });
+
+  return (
```

Comment:
Would it be better to return null when tourOptOut is true to avoid adding these components to the DOM when they are not needed?